packages/@atjson/renderer-commonmark/test/commonmark-test.ts

```diff
@@ -820,7 +820,10 @@ After all the lists
           ]
         });
 
-        expect(CommonmarkRenderer.render(document)).toBe("*a*.b");
+        // preserved here for correct behaviour in case we decide not to throw,
+        // or to implement an `ignoreUnknown` flag or similar.
+        //expect(CommonmarkRenderer.render(document)).toBe("*a*.b");
```

Comment:
github's tslint is mad about this, can you add a space at the start of this comment?

Reply:
I really want the ESLint stuff to be merged 😭

Reply:
lol